source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/MarkDownUnitTest.cpp

```diff
@@ -0,0 +1,318 @@
+#include "stdafx.h"
+#include "CppUnitTest.h"
+#include "MarkDownParser.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+using namespace AdaptiveCards;
+using namespace std;
+
+namespace AdaptiveCardsSharedModelUnitTest
+{
+    TEST_CLASS(EmphasisLeftDelimterTest)
+    {
+        TEST_METHOD(LefDelimiterTest)
```

Comment:
>LefDelimiterTest [](start = 20, length = 16)

Is lef shorthand for left? let's use the complete word.

Reply:
thanks

---
In reply to: [154182001](https://github.com/Microsoft/AdaptiveCards/pull/1003#discussion_r154182001) [](ancestors = 154182001)